pkgs/by-name/do/dokieli/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  mkYarnPackage,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  makeWrapper,
+  nodejs,
+}:
+
+mkYarnPackage rec {
+  pname = "dokieli";
+  version = "0-unstable-2024-09-23";
+
+  src = fetchFromGitHub {
+    owner = "linkeddata";
+    repo = "dokieli";
+    rev = "40ebbc60ba48d8b08f763b07befba96382c5f027";
+    hash = "sha256-lc96jOR8uXLcZFhN3wpSd9O5cUdKxllB8WWCh2oWuEw=";
+  };
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = src + "/yarn.lock";
+    hash = "sha256-TEXCCLFhpwHZJ8zRGsC7J6EwNaFpIi+CZ3L5uilebK4=";
+  };
+
+  packageJSON = ./package.json;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r * $out
+  '';
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postFixup = ''
+    makeWrapper ${nodejs}/bin/npx $out/bin/dokieli \
+      --add-flags serve                            \
+      --chdir $out/deps/dokieli
+  '';
```

Comment:
Running this in an empty environment, I can see that some executable are missing in the path (nodejs for example)
You can use this syntax to fix that:
`--prefix PATH : "${lib.makeBinPath [ A B C  ]}"`

Reply:
Okay, added it. But since dokieli is not an executable, I will need npx to able to wrap it.